charts/opensearch-operator/files/opensearch.opster.io_opensearchclusters.yaml

```diff
@@ -2794,6 +2794,11 @@ spec:
                         - NodePort
                         - LoadBalancer
                         type: string
+                        labels:
```

Comment:
If I'm not wrong this is following the same pattern right ?
https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/docs/userguide/main.md#labels-or-annotations-on-opensearch-nodes

Reply:
Yes, in this case, the labels set on the node pool are propagated to all related resources, including the service. I considered this approach as well, but wanted a more targeted solution, one where labels can be applied exclusively to the service, without requiring the pods to carry them too.